documentation/modules/configuring/proc-config-mirrormaker2-replication.adoc

```diff
@@ -34,14 +34,26 @@ spec:
   version: {DefaultKafkaVersion}
   connectCluster: "my-cluster-target"
   clusters:
-  - alias: "my-cluster-source"
-    bootstrapServers: my-cluster-source-kafka-bootstrap:9092
-  - alias: "my-cluster-target"
-    bootstrapServers: my-cluster-target-kafka-bootstrap:9092
+  - alias: "sourceCluster"
```

Comment:
Please use the original name. This now anyway breaks the whole example because you change the aliases here but the references use the old names.